libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -1033,4 +1033,69 @@ class Collections {
         }
 
     }
+
+    class BinaryOperations {
+
+        @Sample
+        fun unionOfGeneric() {
+            val unionOfArrayAndIterable = arrayOf("Apple", "Banana", "Orange", "Apple").union(
+                listOf("Orange", "Lime")
+            )
+            assertPrints(unionOfArrayAndIterable, "[Apple, Banana, Orange, Lime]")
+            val unionOfTwoIterables = listOf("Apple", "Banana", "Orange", "Apple").union(
```

Comment:
Makes sense to add an empty line before to separate these two cases

Reply:
done